.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: Build
+
+on:
+  push:
+    branches: [staging, trying, main]
+  pull_request:
+    branches: [main]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build examples
+        run: for i in wifi-websockets-workshop; do pushd examples/$i; cargo build --release; popd; done;
```

Comment:
What about doing an `ls` to find directories automatic. And allow for putting a `.skip` file inside, which would skip the directory.

That way you can't forget to add something here, and still opt-out if necessary.